gems/slanger/CVE-2019-1010306.yml

```diff
@@ -0,0 +1,14 @@
+---
+gem: slanger
+url: https://nvd.nist.gov/vuln/detail/CVE-2019-1010306
+date: 2019-07-16
+title: Critical severity vulnerability that affects slanger
+description: 'Slanger 0.6.0 is affected by: Remote Code Execution (RCE). The impact
```

Comment:
Use `description: |` and then just put the description indented on following lines. No need for the quoting.

Reply:
Done.

Just as a note, the description on this one came from the GHSA sync tool. We may want to make some adjustments there as well.